src/experimental/reconciler.ts

```diff
@@ -28,15 +52,22 @@ const hostConfig = {
 		// trigger an immediate render to ensure <Static> children are written to output before they get erased
 		if (rootNode.isStaticDirty) {
 			rootNode.isStaticDirty = false;
-			rootNode.onImmediateRender();
+			if (rootNode.onImmediateRender) {
+				rootNode.onImmediateRender();
+			}
+
 			return;
 		}
 
-		rootNode.onRender();
+		if (rootNode.onRender) {
```

Comment:
```suggestion
		if (typeof rootNode.onRender === 'function') {
```

Reply:
Done in https://github.com/vadimdemedes/ink/pull/264/commits/4c740b887cb1138d5615076d8c9acea0e266ce32